download-maven-plugin/src/main/java/com/googlecode/WGet.java

```diff
@@ -145,26 +145,32 @@
 
   /**
    * Whether to skip execution of Mojo
-   * 
+   *
    * @parameter expression="${download.plugin.skip}" default-value="false"
    */
   private boolean skip;
 
   /**
+   * Whether to check the signature of existing files
+   * @parameter expression="${checkSignature}" default-value="false"
+   */
+  private boolean checkSignature;
```

Comment:
About this change, usually when I'm adding a new feature to an existing package, the "default" value of what I have added is always what we had until now, so the check disabled.
But I don't see any problem to have it always active...